R/extract.R

```diff
@@ -565,11 +638,26 @@ extract.broom <- function(model, ...) {
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{broom} objects
+#'
+#' \code{\link{extract}} method for \code{broom} objects. These objects are
+#' created by the \code{\link[ANY]{broom}} function in the \pkg{ANY} package.
+#'
+#' @inheritParams extract,lm-method
```

Comment:
Can you add manual entries for `model` and `...` in this case instead of inheriting them? For `model`, it should say that it can be any statistical model that is not covered by the `extract` methods in the package but that is contained in the `broom` package. For `...`, it should say that it's currently not in use.

Reply:
sure!